lib/renuo-cms-rails.rb

```diff
@@ -0,0 +1,6 @@
+require 'renuo_cms_rails/version'
+require 'renuo_cms_rails/configuration'
+require 'renuo_cms_rails/cms_helper'
```

Comment:
why did rubocop not complain? ah, because i had to ignore renuo-cms-rails.rb because of the naming... 😢 
done!